spec/css-spec.coffee

```diff
@@ -47,18 +47,41 @@ describe 'CSS grammar', ->
       expect(tokens[8]).toEqual value: 'p', scopes: ["source.css", "meta.selector.css", "entity.name.tag.css"]
 
     describe 'class selectors', ->
```

Comment:
I think it's pretty much covered by `.étendard`, coz if it can match `é`, it surely can match `[a-z]`.